developer-docs-site/static/messageboard_demo/client/README.md

```diff
@@ -0,0 +1,187 @@
+# MessageBoard
+
+### Setup and Run the Demo
+
+Download Aptos Core
+
+```bash
+git clone https://github.com/aptos-labs/aptos-core.git
+cd aptos-core
+./scripts/dev_setup.sh
+source ~/.cargo/env
+```
+
+Install Apto Cli [[doc](https://github.com/aptos-labs/aptos-core/tree/main/crates/aptos)]
+
+```bash
+cargo install --git https://github.com/aptos-labs/aptos-core.git aptos
+```
+
+Run demo using the commands below 
+
+```bash
+cd developer-docs-site/static/messageboard_demo/client
+yarn install
+yarn demo
+```
+
+**This demo shows**
+
+1. write a basic smart contract with move [[source](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/messageboard)]
+2. publish, create and interact with messageboard [[source](https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site/static/messageboard_demo/client)]
+
+A messageboard has an admin who is the creator of the messageboard.
+
+The messageboard has a pinned_post resource. Only participants who have the authority can modify this pinned message resource. Admin can provide this authority to the participants.
+
+Besides the pinned_post resource, anyone can send a message to the messageboard. this will trigger a MessageChangeEvent. Anyone can subscribe to the event stream to read all the messages sent.
+
+## Messageboard Smart Contract
+
+There are two types of messageboards: capability-based messageboard and ACL-based messageboard. The capability-based messageboard is generally more secure [eg: [confused deputy problem](https://en.wikipedia.org/wiki/Confused_deputy_problem)] and also prevents the problem of doing ACL validation against a long list.
+
+The source code can be found here [[source](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/messageboard)]
+
+### Capability-Based Messageboard
```

Comment:
Capability-based Messageboard (in a hyphenated word, the second part is not titlecased).

Reply:
updated